src/App.js

```diff
@@ -1,26 +1,37 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
 
+import CssBaseline from '@material-ui/core/CssBaseline';
+import { MuiThemeProvider, createMuiTheme } from '@material-ui/core/styles';
+import indigo from '@material-ui/core/colors/indigo';
+import deepOrange from '@material-ui/core/colors/deepOrange';
+
+import Login from './pages/Login';
+import Home from './pages/Home';
+
+const theme = createMuiTheme({
+  palette: {
+    primary: indigo,
+    secondary: deepOrange
+  },
+  typography: {
+    useNextVariants: true
+  }
+});
+
+/**
+ * The base of the app, contains all first level routes
+ */
 class App extends Component {
   render() {
     return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            Edit <code>src/App.js</code> and save to reload.
-          </p>
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-        </header>
-      </div>
+      <MuiThemeProvider theme={theme}>
+        <CssBaseline />
+        <Router>
```

Comment:
Create a seperate component <Routes/> where all the routes and component/page imports will be included so the code is more clean